packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -485,16 +523,92 @@ export type TreeNodeFromImplicitAllowedTypes<
 		? NodeFromSchema<FlexListToUnion<TSchema>>
 		: unknown;
 
+/**
+ * This type exists only to be linked from documentation to provide a single linkable place to document some details of
+ * "Input" types and how they handle schema.
+ *
+ * When a schema is used to describe data which is an input into an API, the API is [contravariant](https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science)) over the schema.
+ * (See also, [TypeScript Variance Annotations](https://www.typescriptlang.org/docs/handbook/2/generics.html#variance-annotations)).
+ *
+ * Since these schema are expressed using TypeScript types, its possible for the user of the API to provide non-exact values of these types which has implications that depended ont eh variance.
```

Comment:
```suggestion
 * Since these schema are expressed using TypeScript types, it is possible for the user of the API to provide non-exact values of these types which has implications that depended on the variance.
```